tests/integration/targets/mount/tasks/main.yml

```diff
@@ -126,148 +126,145 @@
 
-- name: Check that we removed the swap2 record
-  assert:
-    that:
-    - swap2_removed['changed']
-    - not swap2_removed_again['changed']
-  when: ansible_system in ('Linux')
+  - name: Check that we removed the swap2 record
+    assert:
+      that:
+      - swap2_removed['changed']
+      - not swap2_removed_again['changed']
+    when: ansible_system in ('Linux')
 
-- name: Create fstab record with missing last two fields
-  copy:
-    dest: /etc/fstab
-    content: '//nas/photo /home/jik/pictures cifs defaults,credentials=/etc/security/nas.creds,uid=jik,gid=users,forceuid,forcegid,noserverino,_netdev
+  - name: Create fstab record with missing last two fields
+    copy:
+      dest: /etc/fstab
+      content: '//nas/photo /home/jik/pictures cifs defaults,credentials=/etc/security/nas.creds,uid=jik,gid=users,forceuid,forcegid,noserverino,_netdev
 
-      '
-  when: ansible_system in ('Linux')
+        '
+    when: ansible_system in ('Linux')
 
-- name: Try to change the fstab record with the missing last two fields
-  mount:
-    src: //nas/photo
-    path: /home/jik/pictures
-    fstype: cifs
-    opts: defaults,credentials=/etc/security/nas.creds,uid=jik,gid=users,forceuid,forcegid,noserverino,_netdev,x-systemd.mount-timeout=0
-    state: present
-  register: optional_fields_update
-  when: ansible_system in ('Linux')
+  - name: Try to change the fstab record with the missing last two fields
+    mount:
+      src: //nas/photo
+      path: /home/jik/pictures
+      fstype: cifs
+      opts: defaults,credentials=/etc/security/nas.creds,uid=jik,gid=users,forceuid,forcegid,noserverino,_netdev,x-systemd.mount-timeout=0
+      state: present
+    register: optional_fields_update
+    when: ansible_system in ('Linux')
 
-- name: Get the content of the fstab file
-  shell: cat /etc/fstab
-  register: optional_fields_content
-  when: ansible_system in ('Linux')
+  - name: Get the content of the fstab file
+    shell: cat /etc/fstab
+    register: optional_fields_content
+    when: ansible_system in ('Linux')
 
-- name: Check if the line containing the missing last two fields was changed
-  assert:
-    that:
-    - optional_fields_update['changed']
-    - ''' 0 0'' in optional_fields_content.stdout'
-    - 1 == optional_fields_content.stdout_lines | length
-  when: ansible_system in ('Linux')
+  - name: Check if the line containing the missing last two fields was changed
+    assert:
+      that:
+      - optional_fields_update['changed']
+      - ''' 0 0'' in optional_fields_content.stdout'
+      - 1 == optional_fields_content.stdout_lines | length
+    when: ansible_system in ('Linux')
 
-- name: Block to test remounted option
-  block:
   - name: Create empty file
-    community.general.filesize:
-      path: /tmp/myfs.img
-      size: 20M
-    when: ansible_system in ('Linux')
+    command: dd if=/dev/zero of=/tmp/myfs.img bs=1048576 count=20
 
   - name: Format FS
     when: ansible_system in ('Linux')
-    community.general.filesystem:
```

Comment:
We should probably keep this as `community.general.filesystem`. They are both the same thing since the one just redirects to the other, but I think `community.general.filesystem` is the documented name and probably what we should use.